config/routes.rb

```diff
@@ -6,10 +6,22 @@
   # root "articles#index"
   root to: 'users#index'
 
+  namespace :api do
+    namespace :v1 do
+      resources :users do
+        resources :posts do
+          resources :comments
+        end
+      end
+    end
+  end
+  
   resources :users, only: [:index, :show] do
     resources :posts, only: [ :index, :new, :create, :show, :destroy] 
     resources :comments, only: [ :create, :destroy]
     resources :likes, only: [ :create ]
   end
 
```

Comment:
- [ ] Please add an API endpoint for authentication, both registration, and login
for example:
* Login endpoint - **POST :** `/login`
* Registration endpoint - **POST :** `/register` or `/signup`

Reply:
@DeliceLydia 
The login and registration endpoints is working very well on my side.
You can log out of the app to check the authentication. Everything is working perfectly